Templates/AzureDevOpsPipeline/README.md

```diff
@@ -47,18 +70,27 @@ Variable | Description
   azureArtifactVersionOption           | Azure artifact version option (when publishing to Azure DevOps Artifacts)
 
+### Supplied Azure pipeline templates 
+
+The directory [templates](templates/) contains additional [Azure pipeline templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops&pivots=templates-includes#parameters-to-select-a-template-at-runtime) that implement actions that are leveraged by the core pipeline template to provide a central control over the tagging and release deployment process. Please review the templates with your Azure administrator and adjust the reference within the [azure-pipelines.yml](azure-pipelines.yml#L32) file.
+
+The tagging process implemented in the template leverages the [AZ REST command of the Azure DevOps CLI](templates/tagging/createReleaseCandidate.yml#L33) to connect with the existing permissions of the pipeline build user to the Azure repository to create the release candidate and the final release tag. Azure DevOps CLI needs to be available on the installed Azure agents.  
+
+### Azure environments
+
+This template uses [Azure deployment jobs](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/deployment-jobs?view=azure-devops) to trigger the deployment on the mainframe and document the deployment of packages to the defined [environments in Azure](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/environments?view=azure-devops). Please create the environments in advance, also consider configuring approvals and checks for environments as gates.
+
+The template allows the development team to add additional environments where they would like to deploy the package to. Please see the [deployment section of the pipeline configuration](azure-pipelines.yml#L187).
+
 ### Installing the private SSH key
 
 To execute actions on the z/OS system, the default communication channel is established via the SSH Service Connection that is provided by Azure.
-To download files from the build workspace on z/OS Unix System Services, it requires to use a `CmdLine` task and write a script.
-SSH communication from a `CmdLine` task does not use the Service Connection.
-To make the agent installation independent of any configurations outside of Azure, a private SSH key is installed as part of the pipeline to connect to the z/OS system through a `CmdLine` task. 
+To download files from the build workspace on z/OS Unix System Services, it requires to use a `CmdLine` task and leverage a script to pull the log files from the mainframe to the Azure agent. The used SSH communication from a `CmdLine` task does not use the Service Connection. To make the agent installation independent of any configurations outside of Azure, a private SSH key is installed as part of the pipeline to connect to the z/OS system through a `CmdLine` task. 
```

Comment:
Minor rephrasing:
To download files from the build workspace on z/OS Unix System Services, the pipeline uses a `CmdLine` task and leverages a script to pull the log files from the mainframe to the Azure agent.